components/ShippingRates/RateTable.vue

```diff
@@ -98,15 +98,15 @@ const emit = defineEmits(['update:modelValue', 'on-update-shipping-rates'])
 const isShippingModalOpened = ref<Boolean>(false)
 
 const hasShipping = ref(props.modelValue)
-const isModalReadOnly = ref(props.readOnly)
+const isModalReadOnly = ref(!props.isClassView)
```

Comment:
will this mutate? if not, then use computed prop is better

Reply:
[🎨 Turn isModalReadOnly to the computed data](https://github.com/likecoin/nft-book-press/pull/130/commits/a998d4536e230b65ee62b6e17bc63059c27afbac)